source/administration/config-settings.rst

```diff
@@ -909,6 +911,22 @@ Set an email for feedback or support requests.
 
 ________
 
+Native App Links
+```````````````````````````
+Mattermost Apps Download Page Link ``"AppDownloadLink": "https://about.mattermost.com/downloads/"`` 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Configurable link to a download page for Native Apps. When a link is present, an option to "Download Mattermost Apps" will be added in the Main Menu so users can find the download page. Leave this field blank to hide the option from the Main Menu. Defaults to a page on about.mattermost.com where users can download the iOS, Android, and Desktop clients. If you are using an `Enterprise App Store <https://docs.mattermost.com/deployment/push.html?highlight=enterprise%20app#push-notifications-and-mobile-devices>`_for your mobile apps, change this link to point to a customized download page where users can find the correct apps.
```

Comment:
Need space after `_` for link to work

![image](https://cloud.githubusercontent.com/assets/177788/17266442/0d7d4982-55ad-11e6-8668-1cac6548806d.png)